helm/charts/grpcserver/base/porter-grpc-deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: porter-grpc-service
+  name: porter-grpc-service
+  namespace: demo
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: porter-grpc-service
+  template:
+    metadata:
+      labels:
+        app: porter-grpc-service
+    spec:
+      volumes:
+        - name: porter-grpc-service-config-volume
+          configMap:
+            name: porter-grpc-service-config
+            items:
+              - key: config
+                path: config.yaml
+      containers:
+        - name: porter-grpc-service
+          image: ghcr.io/getporter/porter-operator:v1.0.0
```

Comment:
this needs to change, can't use the operator for the grpc server

Reply:
Also need the configmap

Reply:
[Do you want this for the gRPC server (the image it produces?)](https://github.com/getporter/porter/blob/main/magefile.go#L85)

Reply:
No this was a placeholder, it now has the getporter grpc-server package.